src/client/script-tag-config.coffee

```diff
@@ -56,9 +56,9 @@ initParams = {
   embed: { default: true, transform: transform.bool },
   autoStart: { default: true, transform: transform.bool },
   accessToken: { default: null }
+  customerId: { default: null }
```

Comment:
Yes ^^ but I do not think that we will have conflict. In fact I do not get why we need to whitelist options.

On Tue, May 12, 2015 at 9:01 AM, Romain Dardour <notifications@github.com>
wrote:

>> @@ -56,9 +56,9 @@ initParams = {
>>    embed: { default: true, transform: transform.bool },
>>    autoStart: { default: true, transform: transform.bool },
>>    accessToken: { default: null }
>> +  customerId: { default: null }
> This seems very specific to Shopify no ?
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/hull/hull-js/pull/189/files#r30130475